platform/ios/src/MGLAnnotationView_Private.h

```diff
@@ -0,0 +1,23 @@
+#import "MGLAnnotationView.h"
+#import "MGLAnnotation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLAnnotationViewDelegate <NSObject>
+
+- (void)annotationView:(MGLAnnotationView *)annotationView didReceiveTapAtPoint:(CGPoint)point;
+
+@end
+
+@interface MGLAnnotationView (Private)
+
+@property (nonatomic, weak) id<MGLAnnotationViewDelegate> delegate;
```

Comment:
Due to this delegate, an MGLAnnotationView instance can only be associated with one MGLMapView at a time. Instead of a dedicated delegate property, could we assume (or assert) that view’s superclass is an MGLMapView? What does MapKit do when you try to insert an MKAnnotationView into a generic UIView’s subview list?